docs/EXAMPLES.md

```diff
@@ -27,17 +27,17 @@ const protractorHelper = require("protractor-helper");
 describe("foo", () => {
     it("bar", () => {
         // ...
-        // Here all `protractor-helper` method use the default timeout.
+        // Here all the protractor-helper methods use the default timeout.
```

Comment:
Sorry, but the '`'s can stay.

Reply:
The '`' is a syntax highlighting, and don't work inside other syntax highlighting. I see no need to keep them.

Reply:
Ok.